src/main/js/helpers/axis.js

```diff
@@ -205,26 +221,67 @@ const getAxesScale = (axis, scale, config) => {
             config.axis.y2.domain.upperLimit - config.axis.y2.domain.lowerLimit
         );
 
+    if (
+        config.suppressTrailingZeros &&
```

Comment:
Why did not require a Y-axis type default check here like we added for the x-axis?

Reply:
I have not added "axis-type == default" check for y-axis since there can be only number based axes values for y axis. The "axis-type == default" check is required for x-axis since there can be 2 types of axes values like time series or number based values for x-axis.

Added comment for the conditional statement [here](https://github.com/JananiGunasekaran/carbon-graphs/commit/7d888e61d33050d9b0099e5d57adac17c5cb1b17).